.gitignore

```diff
@@ -2,7 +2,6 @@
 .bundle
 .sass-cache
 _site/
-Gemfile.lock
```

Comment:
why was this even in gitignore so far?

Reply:
very good question; last change was 5 years ago so i suspect teh answer has been lost to time

Reply:
wait no this should be here right? this file is created when we build the jekyll app locally no?

Reply:
not sure what you mean here; this is the equivalent of `package.lock`

Reply:
yea is a lockfile, shld be commited